src/tests/test_syncify.py

```diff
@@ -1,6 +1,44 @@
 import pytest
 
 
+@pytest.mark.xfail_browsers(safari="No JSPI on Safari", firefox="No JSPI on firefox")
+def test_syncify_create_task(selenium):
```

Comment:
@hoodmane I replaced second test with new one
But now I'm curious about using `asyncio.gather()` with `syncify()`. Is it possible?

I tried using wrapper function, and it's working, but it's workaround, not "from the box" solution
Example with wrapper function:
```python
await pyodide.runPythonSyncifying(`
    import asyncio

    async def test(x):
        await asyncio.sleep(0.1 * x)
        return x + 1

    async def better_gather(*coros):
        return await asyncio.gather(*coros)

    res = asyncio.create_task(better_gather(test(5), test(3), test(10))).syncify()
    print(*res)
`);
```

Reply:
>  Maybe will be better to add test with exception inside Python (and use pytest.raises), instead of this one?

The current test looks good but why not keep both the one with the exception and the original test you wrote?

Reply:
>  But now I'm curious about using asyncio.gather() with syncify(). Is it possible?

Thanks for pointing this out. Could you open an issue about it?

Reply:
> > Maybe will be better to add test with exception inside Python (and use pytest.raises), instead of this one?
> 
> The current test looks good but why not keep both the one with the exception and the original test you wrote?

But I kept original one — `test_syncify_create_task`. I replaced only second one — `test_syncify_ensure_future`
Or do you think, that test with `ensure_future` was better?

Reply:
> > But now I'm curious about using asyncio.gather() with syncify(). Is it possible?
> 
> Thanks for pointing this out. Could you open an issue about it?

Yeah, thanks, I will open an issue :)

Reply:
> But I kept original one — test_syncify_create_task. I replaced only second one

Sorry I think I misread the diff.